From: Jie Liu <liujie01@kylinos.cn>
Date: Thu, 5 Sep 2024 13:02:20 +0800
Subject: remove support for ukui_surface_position property

Now you can you qt setGeometry or move to set window position, this
property is not necessary.
---
 src/ukuishellsurface.cpp | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/ukuishellsurface.cpp b/src/ukuishellsurface.cpp
index 10c52c7..052fb8e 100644
--- a/src/ukuishellsurface.cpp
+++ b/src/ukuishellsurface.cpp
@@ -169,7 +169,7 @@ void UkuiShellSurface::setContentGeometry(const QRect &rect)
 
 void UkuiShellSurface::setWindowPosition(const QPoint &position)
 {
-    if (m_toplevel)
+    if (m_toplevel && m_configured)
         set_position(position.x(), position.y());
 }
 
@@ -458,9 +458,6 @@ void UkuiShellSurface::slotWindowPropertyChanged(const char *name)
     } else if (strcmp(name, "ukui_surface_open_under_cursor") == 0) {
         const auto point = window->property(name).toPoint();
         open_under_cursor(point.x(), point.y());
-    } else if (strcmp(name, "ukui_surface_position") == 0) {
-        const auto point = window->property(name).toPoint();
-        set_position(point.x(), point.y());
     } else if (strcmp(name, "ukui_surface_skip_taskbar") == 0) {
         setSkipTaskbar(window->property(name).toBool());
     } else if (strcmp(name, "ukui_surface_skip_switcher") == 0) {
